packages/frontend/src/features/sqlRunner/store/barChartSlice.ts

```diff
@@ -0,0 +1,77 @@
+import { ChartKind, type BarChartConfig } from '@lightdash/common';
+import type { PayloadAction } from '@reduxjs/toolkit';
+import { createSlice } from '@reduxjs/toolkit';
+import { setInitialResultsAndSeries, setSaveChartData } from './sqlRunnerSlice';
+
+const initialState: { config: BarChartConfig | undefined } = {
+    config: undefined,
+};
+
+export const barChartSlice = createSlice({
```

Comment:
```suggestion
export const barChartConfigSlice = createSlice({
```

Reply:
And if you could rename the file to `barChartConfigSlice` so everything matches 👍